scripts/local_k8s/unittests/test_cli.py

```
@@ -53,56 +44,110 @@ def _unwrap(name: str, args: list[str]) -> tuple[str, list[str]]:
-FAKE_TOOL_NAMES = (
-    "docker",
-    "podman",
-    "helm",
-    "k3d",
-    "kind",
-    "kubectl",
-    "uv",
-    "env",
-    "systemd-run",
-)
+FAKE_TOOL_NAMES = "docker podman helm k3d kind kubectl uv env systemd-run".split()
```

Ruff (0.16.0)

[warning] Consider using a list literal instead of `str.split`

Replace with list literal

(SIM905)